ArduCopter/fence.cpp

```diff
@@ -5,13 +5,13 @@
 #if AP_FENCE_ENABLED
 
 // fence_check - ask fence library to check for breaches and initiate the response
-// called at 1hz
+// called at 25hz
```

Comment:
this is being debated....

Reply:
Ah that snuck in accidentally - removed now, thanks!